Calendar/Get-RBASummary.ps1

```diff
@@ -90,39 +90,41 @@ function EvaluateCalProcessing {
-    Write-Host "`t AllowDistributionGroup:         "$RbaSettings.AllowDistributionGroup
-    Write-Host "`t AllowMultipleResources:         "$RbaSettings.AllowMultipleResources
-    Write-Host "`t MaximumDurationInMinutes:       "$RbaSettings.MaximumDurationInMinutes
-    Write-Host "`t MinimumDurationInMinutes:       "$RbaSettings.MinimumDurationInMinutes
-    Write-Host "`t AllowRecurringMeetings:         "$RbaSettings.AllowRecurringMeetings
-    Write-Host "`t ScheduleOnlyDuringWorkHours:    "$RbaSettings.ScheduleOnlyDuringWorkHours
-    Write-Host "`t ProcessExternalMeetingMessages: "$RbaSettings.ProcessExternalMeetingMessages
-    Write-Host "`t BookingWindowInDays:            "$RbaSettings.BookingWindowInDays
-    Write-Host "`t ConflictPercentageAllowed:      "$RbaSettings.ConflictPercentageAllowed
-    Write-Host "`t MaximumConflictInstances:       "$RbaSettings.MaximumConflictInstances
-    Write-Host "`t MaximumConflictPercentage:      "$RbaSettings.MaximumConflictPercentage
-    Write-Host "`t EnforceSchedulingHorizon:       "$RbaSettings.EnforceSchedulingHorizon
-    Write-Host "`t SchedulingHorizonInDays:        "$RbaSettings.SchedulingHorizonInDays
+    Write-Host -ForegroundColor Cyan "`t Setting                          Value"
+    Write-Host -ForegroundColor Cyan "`t -------------------------------  -----------------------------"
+    Write-Host -ForegroundColor Cyan "`t AllowConflicts:                 "$RbaSettings.AllowConflicts
+    Write-Host -ForegroundColor Cyan "`t AllowDistributionGroup:         "$RbaSettings.AllowDistributionGroup
+    Write-Host -ForegroundColor Cyan "`t AllowMultipleResources:         "$RbaSettings.AllowMultipleResources
+    Write-Host -ForegroundColor Cyan "`t MaximumDurationInMinutes:       "$RbaSettings.MaximumDurationInMinutes
+    Write-Host -ForegroundColor Cyan "`t MinimumDurationInMinutes:       "$RbaSettings.MinimumDurationInMinutes
+    Write-Host -ForegroundColor Cyan "`t AllowRecurringMeetings:         "$RbaSettings.AllowRecurringMeetings
+    Write-Host -ForegroundColor Cyan "`t ScheduleOnlyDuringWorkHours:    "$RbaSettings.ScheduleOnlyDuringWorkHours
+    Write-Host -ForegroundColor Cyan "`t ProcessExternalMeetingMessages: "$RbaSettings.ProcessExternalMeetingMessages
+    Write-Host -ForegroundColor Cyan "`t BookingWindowInDays:            "$RbaSettings.BookingWindowInDays
+    Write-Host -ForegroundColor Cyan "`t ConflictPercentageAllowed:      "$RbaSettings.ConflictPercentageAllowed
+    Write-Host -ForegroundColor Cyan "`t MaximumConflictInstances:       "$RbaSettings.MaximumConflictInstances
+    Write-Host -ForegroundColor Cyan "`t MaximumConflictPercentage:      "$RbaSettings.MaximumConflictPercentage
+    Write-Host -ForegroundColor Cyan "`t EnforceSchedulingHorizon:       "$RbaSettings.EnforceSchedulingHorizon
+    Write-Host -ForegroundColor Cyan "`t SchedulingHorizonInDays:        "$RbaSettings.SchedulingHorizonInDays
```

Comment:
Don't like this format. We could change this up a bit, you can create a function that handles this based off an object that you created, it would make it cleaner to read. If that is something you want to do and don't know how let me know.

Reply:
I have improved the formatting, but if you were talking something different, please send me a pointer to an example.

Reply:
An example would be something like this this file:

https://github.com/microsoft/CSS-Exchange/blob/main/Search/Troubleshoot-ModernSearch/Write/WriteHelpers.ps1

However, I think for now we can leave it as is and I will create an issue to create a shared function that we can use easier.

Reply:
For reference #1773